wiki/Tournaments/GO/3/en.md

```diff
@@ -8,7 +8,7 @@ tags:
 
 ![GO 3 Logo](img/logo.png)
 
-The **Game\_Osu Tourney 3** (**GO 3**) is a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) and ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), and supported by ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). It is the third installment of the Game\_Osu Tourney.
+The **Game_Osu Tourney 3** (***GO 3***) was a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) and ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), and supported by ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). It was the third installment of the Game_Osu Tourney.
```

Comment:
not escaping the `_` here?

Reply:
there's nothing to escape, underscore doesn't do anything here

Reply:
So change it or no?

Reply:
Leave it as is.